public/api-merged.json

```diff
@@ -6469,6 +6469,145 @@
         }
       }
     },
+    "/provisioning/contact-points": {
```

Comment:
I don't know how this change got here. Does anyone understand this?

Reply:
I think you did run `make swagger-api-spec` which is new and didn't was applied for my PR yet, as it wasn't existing back then. Feel free to delete this part, I can add it in a separate PR.